frontend/package.json

```diff
@@ -68,6 +68,7 @@
     "@types/uuid": "^8.3.0",
     "@typescript-eslint/eslint-plugin": "^5.4.0",
     "@typescript-eslint/parser": "^5.4.0",
+    "@vicons/ionicons5": "^0.12.0",
```

Comment:
And it looks like we have to add another package `@vicons/ionicons5` when I only upgrade `naive-ui`. @LiuJi-Jim 

![image](https://user-images.githubusercontent.com/24653555/187146284-8b54b256-c0ed-4ed1-8039-d90308913c16.png)